### DISCUSSION TITLE
How can I create the .scr file myself?

### DISCUSSION THREAD

**Discussion by mathiasmadsen** [OrangeJedi/Aerial#185] [Q&A]
Thanks for a great project. This could very well be usable for my needs.

While the .exe version seems to work as intended, I would like to be able to create the .scr file, as I have made some local changes to the project.

I can see that I am able to unpack the content of the aerial.scr file from the GitHub release, but I have no success in repacking it to make it work? I have also tried renaming the .exe file, but this also doesn't seem to work.

Can anyone help me out? :)

**Comment by aerickson** [comment]
Digging through the original repo I found:

https://github.com/cDima/Aerial/blob/226ecad4709bf81b43564c9b653d868c464fc50b/ScreenSaver/ScreenSaver.csproj#L220

`copy /Y "$(TargetDir)$(TargetName).exe" "$(TargetDir)\$(TargetName).scr`

Could it be that simple?
